.github/workflows/ci.yml

```diff
@@ -10,6 +10,22 @@ jobs:
   build:
 
     runs-on: macos-11
+    env:
+      SCHEME: "swift-user-defaults"
+      EXAMPLE_SCHEME: "Example"
+      WORKSPACE: "Example/Example.xcworkspace"
```

Comment:
Since we only use these two variables once, maybe we can just type them inline? WDYT?